test/mpi/maint/gentests_dtp.sh

```diff
@@ -158,6 +158,38 @@ while read -r line ; do
                     echo "${testname} $procs arg=-type=${type} arg=-sendcnt=${sendcount} arg=-recvcnt=${recvcount} arg=-seed=$seed arg=-testsize=${gputestsize} ${other_args} arg=-sendmem=device arg=-recvmem=device $timelimit" >> ${builddir}/${testdir}/testlist.gpu
                     seed=$((seed + 1))
                 done
+            elif [ $testdir = "rma" ] ; then
+                if grep -q ^lock <<<"$testname" ; then # lock_x_dt.c uses MPI_Get_accumulate, so we must specify resultmem
```

Comment:
This code logic seems very fragile.  Can you at least check for the full file name?  Or better yet, create separate lists for tests that use Get_accumulate and tests that don't?

Reply:
Will create a list for get_accumulate tests.